policy/policy.go

```diff
@@ -257,3 +343,15 @@ type PoliciesList []StagedPolicies
 func (l PoliciesList) IsDefault() bool {
 	return len(l) == 1 && l[0].IsDefault()
 }
+
+// WithDefaultAggregation updates the PoliciesList with default aggregation types
+func (l PoliciesList) WithDefaultAggregation() PoliciesList {
+	for _, sp := range l {
+		for j := range sp.policies {
+			if !sp.policies[j].compressedAggTypes.IsDefault() {
```

Comment:
How is this function used? Seems like we are overriding the aggregation types here, which is kinda strange.